docs/testing/testplan.md

```diff
@@ -0,0 +1,427 @@
+2. Open the extension and make a one-time payment of an amount that falls within your remaining balance shown in the extension.
+3. Visit a non-web monetized website, such as your favorite search engine. Open the extension to observe its available options.
+
+**Expected results**:  
+ | Web monetized websites | Non monetized websites |
+| :---------------------------- | :--------------------------------------------------- |
+| Extension icon: active (full color), with a green tick | Extension icon: active (full color), but with a red X |
+| Opening the extension displays: <ul><li>**Rate of pay slider**: rate of pay and currency on the left, the remaining balance of the extension is on the right side, and it decreased by the value of the one-time payment</li><li>**Amount**: the one-time payment amount field resets to zero</li><li>**“Send now” button**: clicking the button to send a one-time payment changes the text to “Payment successful” for a few seconds, then the text defaults back to “Send now” </li></ul> | Opening the extension displays: <ul><li>This website is not monetized</li></ul> |
+
+##### Pay one-time when extension funds are insufficient
+
+**Test ID**: 4  
+**Description**: Send a one-time payment to a monetized website that is greater than the remaining balance of the extension, while the wallet has sufficient funds  
+**Risk**: R3 (security of funds) and R4 (control my payments)  
+**Preconditions**:
+
+1. The extension is connected to your wallet.
+2. Your extension has insufficient funds. The extension’s remaining balance is lower than the one-time payment you will make.
+3. Your wallet has sufficient funds. Your balance is greater or equal to the one-time payment you will make.
+
+**Steps**:
+
+1. Visit a monetized website. The [Test Data](#Test-Data) section lists monetized websites that you can visit.
+2. Open the extension and make a one-time payment of an amount that falls within your remaining balance shown in the extension.
+3. Visit a non-web monetized website, such as your favorite search engine. Open the extension to observe its available options.
+
+**Expected results**:  
+| Web monetized websites | Non monetized websites |
+| :------------------------------------------------------- | :------------------------------------------------------ |
+| Extension icon: active (full color), with a green tick | Extension icon: active (full color), but with a red X |
+| Opening the extension displays: <ul><li>**A slider**: with the hourly rate of pay and currency on the left, and the remaining balance of the extension’s authorized amount on the right side.</li><li>**The “Send now” button**: attempting to make a one-time payment that is greater than the remaining balance fails with an error: `Not enough funds to facilitate payment`</li></ul> | Opening the extension displays: <ul><li>This website is not monetized.</li></ul> |
```

Comment:
We updated some copy in https://github.com/interledger/web-monetization-extension/commit/dfb1229cde073801a3435c6c5829ab5c9ff0b834. Not sure if we want latest copy here or you want to update as a follow-up.

```suggestion
| Opening the extension displays: <ul><li>**A slider**: with the hourly rate of pay and currency on the left, and the remaining balance of the extension’s authorized amount on the right side.</li><li>**The “Send now” button**: attempting to make a one-time payment that is greater than the remaining balance fails with an error: `Insufficient funds to complete the payment.`</li></ul> | Opening the extension displays: <ul><li>This website is not monetized.</li></ul> |
```